v2/internal/attacktechniques/gcp/privilege-escalation/impersonate-service-accounts/main.tf

```
@@ -24,7 +24,7 @@ resource "random_string" "suffix" {
 // Create N service accounts
 resource "google_service_account" "service_account" {
   count       = local.num-service-accounts
-  account_id  = format("stratus-red-team-%s", random_string.suffix[count.index].result)
+  account_id  = "stratus-red-team-impersonate-sa-${random_string.suffix[count.index].result}"
```

I kind of liked the `format` way of doing it

```suggestion
  account_id  = format("stratus-red-team-impersonate-sa-%s", random_string.suffix[count.index].result)
```